### PR TITLE
Docs: Run swarm using the docker official image in the examples.

### PR DESCRIPTION
**Pull request by aluzzardi**
Signed-off-by: Andrea Luzzardi <aluzzardi@gmail.com>

### CODE REVIEW

**Comment by aluzzardi** [comment]
/cc @SvenDowideit 

This is not working yet but should very soon.

**Comment by vieux** [comment]
It's good to be reviewed, the image is live https://registry.hub.docker.com/_/swarm/

**Comment by vieux** [comment]
ping @SvenDowideit

**Comment by SvenDowideit** [comment]
YAY, LGTM. using docker to deal with logs and other fun makes so much sense.

File: README.md [hunk 1]
```diff
@@ -17,17 +17,28 @@ in more powerful backends, like `Mesos`, for large scale production deployments.
 
 ## Installation
 
-###1 - Download and install the current source code.
+###1 - Docker image.
+The easiest way to get started with Swarm is to use the
+[official Docker image](https://registry.hub.docker.com/_/swarm/).
+
+```sh
+docker pull swarm
```
Comment (bfirsh) [inline README.md]: Maybe just `docker run swarm` and it'll print usage?

Comment (aluzzardi) [inline README.md]: We want people to upgrade swarm eventually, getting them familiar with `pull` seemed the way to go

**Comment by vieux** [comment]
LGTM

**Pull request merged**